packages/provwasm-std/src/types/cosmos/app/module/v1alpha1.rs

```diff
@@ -11,5 +11,5 @@ use provwasm_proc_macro::{CosmwasmExt, SerdeEnumAsInt};
     ::schemars::JsonSchema,
     CosmwasmExt,
 )]
-#[proto_message(type_url = "/cosmos.authz.module.v1.Module")]
+#[proto_message(type_url = "/cosmos.app.module.v1alpha1.Module")]
```

Comment:
@kwtalley there's many changes like this that I wasn't expecting as it seems like this change is going backwards? Were there manual changes to these files or were they last generated with a different version of cosmos or something?